### PR TITLE
added author page #6433

### PR DESCRIPTION
**Pull request by gautampanday**
Add the author page and provide the link at autho of blog page

### CODE REVIEW

File: theme/templates/author.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{% set SITE_ROOT = '../' %}
```
Comment (GauravButola) [inline theme/templates/author.html:1]: Also change page title and create an 'Authors' page.

**Comment by GauravButola** [comment]
LGTM.

File: theme/templates/authors.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<section class="section-space">
+ {% for author, articles in authors|sort %}
+ <div class="col-sm-offset-4 col-sm-4">
+  <ul class="list-group">
+    <li class="list-group-item">
+      <span class="badge">{{ articles|count }}</span>
+      <a href="{{ SITEURL }}/{{ author.url }}">{{ author }}</a>
+    </li>
+  </ul>
+</div>
```
Comment (tarunbhardwaj) [inline theme/templates/authors.html]: indent

File: theme/templates/author.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+{% extends 'index.html' %}
+{% block pagetitle %}
+<section class="app-banner">
+  <div class="container">
+    <ol class="breadcrumb remove-background">
+      <li><a class="white-text-color" href="{{ SITE_ROOT }}index.html">{% trans %} Home {% endtrans %}</a></li>
```
Comment (tarunbhardwaj) [inline theme/templates/author.html]: don't include trailing space in trans

File: theme/templates/index.html [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {% set SITE_ROOT = SITE_ROOT or '../' %}
 {% extends 'base.html' %}
 {% block main %}
+{% block pagetitle %}
```
Comment (tarunbhardwaj) [inline theme/templates/index.html:4]:
``` jinja
 <section class="blog-heading">
   <div class="container">
     <h1 class="white-text-color margin-0">{% block pagetitle %}{% trans %}Blog{% endtrans %}{% endblock pagetitle %}</h1>
     <ol class="breadcrumb remove-background padding-0 margin-0">
       <li><a class="white-text-color" href="{{ SITE_ROOT }}index.html">{% trans %}Home{% endtrans %}</a></li>
       {% block breadcrumb %}
         <li class="white-text-color">{% trans %}Blog{% endtrans %}</li>
       {% endblock breadcrumb %}
     </ol>
   </div>
 </section>
```

File: theme/templates/index.html [hunk 5]
```diff
@@ -10,50 +11,54 @@ <h1 class="white-text-color margin-0">{% trans %}Blog{% endtrans %}</h1>
...
+                <table border="0" cellpadding="0" cellspacing="0">
+                  <tr>
+                    <td class="blog-month">
+                      {{ article.date|strftime('%B') }}
+                    </td>
+                  </tr>
+                  <tr>
+                    <td class="blog-time">
+                     <h4 class="margin-0"><strong>{{ article.date|strftime('%d') }}</strong></h4>
+                   </td>
```
Comment (tarunbhardwaj) [inline theme/templates/index.html]: fix indentation from here

**Comment by tarunbhardwaj** [comment]
Do update title block as defined by base.html

**Pull request closed without merging**